src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -398,4 +398,6 @@ public net.minecraft.world.level.newbiome.layer.Layers m_76729_(JLnet/minecraft/
 protected net.minecraft.world.level.portal.PortalForcer f_77648_ # level
 public net.minecraft.world.level.storage.LevelResource <init>(Ljava/lang/String;)V # constructor
 private-f net.minecraft.world.level.storage.loot.LootPool f_79028_ # rolls
-private-f net.minecraft.world.level.storage.loot.LootPool f_79029_ # bonusRolls
\ No newline at end of file
+private-f net.minecraft.world.level.storage.loot.LootPool f_79029_ # bonusRolls
+public net.minecraft.tags.TagManager$LoaderInfo
+public net.minecraft.tags.TagManager$LoaderInfo <init>(Lnet/minecraft/tags/StaticTagHelper;Ljava/util/concurrent/CompletableFuture;)V
```

Comment:
These should be alphabetized to the right spot in the file with the empty line removed, then run checkAts.